_posts/2022-04-19-small-development-grant.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "NumFOCUS small development grant: adding complex number support to JuMP"
+date:   2022-04-19
+categories: [announcements]
+---
+
+The [JuMP Steering Committee](/pages/governance/#steering-committee) is pleased
+to announce that we have received a
+[small development grant](https://numfocus.org/programs/small-development-grants)
+from [NumFOCUS](https://numfocus.org) to add complex number support to JuMP.
+
+Currently, JuMP and MathOptInterface are limited to formulating optimization
+problems with real-valued decision variables. However, complex numbers appear in
+a variety of industrial optimization problems. The most important of these is
```

Comment:
Maybe not under-rate other aplications of complex numbers...
"One of the most important..."
"One of the most famous..."